temporal-sdk/src/test/java/io/temporal/workflow/TerminatedWorkflowTest.java

```diff
@@ -53,24 +59,49 @@ public void testShouldReturnQueryResultAfterWorkflowTimeout() {
             .toBuilder()
             .setWorkflowRunTimeout(Duration.ofSeconds(1))
             .build();
-    TestTraceWorkflow client =
+    TestTraceWorkflow workflow =
         testWorkflowRule.getWorkflowClient().newWorkflowStub(TestTraceWorkflow.class, options);
 
     Assert.assertThrows(
         "Workflow should throw because of timeout",
         WorkflowFailedException.class,
-        () -> client.execute(SDKTestWorkflowRule.useExternalService));
+        workflow::execute);
```

Comment:
I had to clean this up as a part of this PR, because TraceWorkflow for some reason accepting `useExternalService` as a parameter just doesn't make sense and it's a part of this test class.